File: diet/src/main/java/com/example/diet/Domain/Article.java
package com.example.diet.Domain;


import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Article {
    /**
     * 文章id
     */
    private long article_id;
    /**
     * 内容来源
     */
    private String article_src;
    /**
     * 文章标题
     */
    private String article_title;
    /**
     * 文章html
     */
    private String content;
    /**
     * 封面图url
     */
    private String pic_url;
}

File: frontend/DayDiet_google/app/src/main/java/com/example/daydiet/model/Domain/Article.kt
package com.example.daydiet.model.entity

data class Article(
    val article_id : Long,   //
    val content : String,    //文章html
    val pic_url : String,    //封面图片url
    val article_title : String,   //文章标题
    val article_src : String     //内容来源
)


// 获取推荐文章信息
data class ArticleResponse(var data:List<Article> ): BaseResponse() {}
